services/drupal/web/modules/custom/epa_core/epa_core.deploy.php

```diff
@@ -433,18 +433,157 @@ function epa_core_deploy_0004_set_card_field_default_values(&$sandbox) {
           \Drupal::logger('epa_core')->error("Error inserting record into $field_table table: " . $e->getMessage());
         }
       }
-    else {
-      // Record does exist but is null, run update.
-      try {
-        $database->update($field_table)->fields([
-          'field_flag_card_alignment_value' => 'default'
-        ])->execute();
-      }
-      catch (\Exception $e) {
-        \Drupal::logger('epa_core')->error("Error updating record into $field_table table: " . $e->getMessage());
+      else {
+        // Record does exist but is null, run update.
+        try {
+          $database->update($field_table)->fields([
+            'field_flag_card_alignment_value' => 'default'
+          ])->execute();
+        }
+        catch (\Exception $e) {
+          \Drupal::logger('epa_core')->error("Error updating record into $field_table table: " . $e->getMessage());
+        }
+
       }
+    }
+  }
+}
+
+/**
+ * Setting notification_opt_in flag for node author, author of the latest
+ * revision, or author of the current revision
+ */
+function epa_core_deploy_0005_set_watch_flag(&$sandbox) {
+  if (!isset($sandbox['total'])) {
+    // Get all nodes.
+    $nodes = \Drupal::database()->query(
+      "SELECT node.nid,
+         latest.revision_uid,
+         node.uid,
+         current.revision_uid
+FROM node_revision AS latest
+LEFT JOIN node_field_data AS node
+    ON latest.nid = node.nid
+LEFT JOIN node_revision AS current
+    ON node.vid = current.vid
+WHERE latest.vid IN
+    (SELECT MAX(vid)
+    FROM node_revision
+    GROUP BY  nid);")
+      ->fetchAll();
+
+    $sandbox['total'] = count($nodes);
```

Comment:
I'd encourage you to change the query to a count query to avoid transferring this huge amount of data to PHP then having to do the count in PHP.  Just have the DB count it for you.